### PR TITLE
Aries share text

### PR DESCRIPTION
**Pull request by dcvespa**

### CODE REVIEW

File: src/AriesSDK/AriesSDK.js [hunk 1]
```diff
@@ -191,6 +191,13 @@ class AriesSDK {
 		})
 	}
 
+	launchShareService(content) {
+		return new Promise((resolve, reject) => {
```
Comment (basiclines) [inline src/AriesSDK/AriesSDK.js]: We should check whenever `content` is empty or not and then throw a `console.warn` like other methods do

File: src/AriesSDK/AriesSDK.js [hunk 2]
```diff
@@ -191,6 +191,13 @@ class AriesSDK {
...
+			this._subscribe(this._events.agenda, resolve, reject)
```
Comment (basiclines) [inline src/AriesSDK/AriesSDK.js]: Why are we using the `agenda` event namespace? We should have a new event namespace for this method.
We can call it`share` i.e.

**Comment by basiclines** [comment]
BTW we have a PR template, it didn't work?

File: src/AriesSDK/AriesSDK.js (hunk 1 above)
Comment (dcvespa) [reply src/AriesSDK/AriesSDK.js]: About _connectBridgeResponses, I'll check the specs with Paulo and add to the PR.

**Comment by dcvespa** [comment]
> BTW we have a PR template, it didn't work?

I believe I didn't see that by distraction. Sorry.

**Pull request closed without merging**